### PR TITLE
Add endpoints to fetch gcp networks

### PR DESCRIPTION
**Pull request by kgroschoff**
**What this PR does / why we need it**:
Added endpoints to fetch a list of gcp networks

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Related to https://github.com/kubermatic/dashboard-v2/issues/1519

**Special notes for your reviewer**:
/

**Does this PR introduce a user-facing change?**:

```release-note
Added endpoints to fetch a list of gcp networks
```

### CODE REVIEW

File: api/pkg/api/v1/types.go [hunk 1]
```diff
@@ -275,6 +275,20 @@ type GCPZone struct {
 // swagger:model GCPZoneList
 type GCPZoneList []GCPZone
 
+// GCPNetworkList represents an array of GCP networks.
+// swagger:model GCPNetworkList
+type GCPNetworkList []GCPNetwork
+
+// GCPNetwork represents a object of GCP machine size.
```
Comment (maciaszczykm) [inline api/pkg/api/v1/types.go]: Typo.

Comment (kgroschoff) [reply api/pkg/api/v1/types.go]: Fixed

**Pull request merged**